src/scales/index.ts

```diff
@@ -100,3 +103,23 @@ export function isTransformable(scale: any): scale is TransformableScale {
   return (scale instanceof QuantitativeScale ||
   scale instanceof Category);
 }
+
+/*
+ * Whether scale should be comparable intuitively.
+ */
+export function isNotComparable(scale1: any, scale2: any): boolean {
```

Comment:
We shouldn't structure it like this; the methods on the scales themselves should define how the comparison works. How did the comparison work correctly pre-2.2.0? What did we change that introduced the bug? We should tackle it there instead